ppr-ui/src/components/common/UpdatedBadge.vue

```diff
@@ -1,14 +1,16 @@
 <template>
-  <v-chip
-    v-if="hasChanges"
-    id="updated-badge-component"
-    xSmall
-    variant="flat"
-    color="primary"
-    :data-test-id="`${action}-badge`"
-  >
-    <b>{{ action }}</b>
-  </v-chip>
+  <div>
+    <v-chip
+      v-if="hasChanges"
+      id="updated-badge-component"
+      xSmall
+      variant="flat"
+      color="primary"
+      :data-test-id="`${action.toLocaleLowerCase()}-badge`"
+    >
+      <b>{{ action.toUpperCase() }}</b>
```

Comment:
Works fine but also redundant. Sort of just removes flexibility and adds more code, so not sure the rationale.

Reply:
Liz confirmed it will always be uppercase. Yes it removes the flexibility but makes it cleaner. I can revert, no biggie.

Reply:
Not sure adding a function to convert it to upper and a function to convert to lower is cleaner, is all.

Reply:
That's so we don't have to think about what to pass as a prop, '`amended'` or `'Amended'`

Reply:
Not sure i agree with the argument, we have enums or configuration, you only have to type out 'AMENDED' once. 
Adding not one but two functions just to remove flexibility is a stance i'm just not following.

Reply:
Not once, but a couple of times (components, tests). I updated, you can see in last commit.